lib/ConstExtract/ConstExtract.cpp

```diff
@@ -120,77 +120,71 @@ parseProtocolListFromFile(StringRef protocolListFilePath,
   return true;
 }
 
-static std::string extractLiteralOutput(Expr *expr) {
-  std::string LiteralOutput;
-  llvm::raw_string_ostream OutputStream(LiteralOutput);
-  expr->printConstExprValue(&OutputStream, nullptr);
-
-  return LiteralOutput;
-}
-
-static std::shared_ptr<CompileTimeValue>
-extractPropertyInitializationValue(VarDecl *propertyDecl) {
-  auto binding = propertyDecl->getParentPatternBinding();
-  if (binding) {
-    auto originalInit = binding->getOriginalInit(0);
-    if (originalInit) {
-      auto literalOutput = extractLiteralOutput(originalInit);
+static std::shared_ptr<CompileTimeValue> extractCompileTimeValue(Expr *expr) {
+  if (expr) {
```

Comment:
Were you seeing nullptrs passed into this function?

Reply:
This is more of a precautionary construct, to avoid bad dereferences if `nullptr` were to get passed in, now or in the future.